docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+services:
+
+  backend:
+    image: python:3.5
+    command: python3 /app/refactor_me.py
+    environment:
+      - PYTHONUNBUFFERED=1
+    volumes:
+      - ./:/app
+
+  flake:
+    image: flake:anton
+    command: bash -c "cd /home/docker && gulp"
```

Comment:
why not to use `working_dir` option?

Reply:
good point :) That's why i like PR reviews.